modules/javafx.controls/src/test/java/test/javafx/scene/control/ListViewTest.java

```diff
@@ -2518,4 +2518,46 @@ public static void verifyListViewScrollTo(ListView listView, int listViewHeight,
 
     }
 
+    @Test
+    public void fixListViewCrash_JDK_8303680() {
+        final ListView<String> listView = new ListView<>();
+
+        // add 100 entries
+        listView.setPrefSize(200, 200);
+        for (int i = 0; i < 100; i++) {
+            listView.getItems().add("Item " + i);
+        }
+        StageLoader sl = new StageLoader(new VBox(listView, new Button()));
+
+        // pulse
+        Toolkit.getToolkit().firePulse();
+
+        // get virtual flow
+        VirtualFlow vf = VirtualFlowTestUtils.getVirtualFlow(listView);
+
+        // scroll to 50 and scroll 1 pixel
+        vf.scrollTo(50);
+        vf.scrollPixels(1);
```

Comment:
this is a rather artificial test.  
is it possible to reproduce the condition using control's public APIs?

Reply:
This is the public API of VirtualFlow. I don't know whether it's possible with the API of the Control - But the API of the VirtualFlow should be stable too, right?

Reply:
You are right, it is public API.  One can't get directly to it, but it is possible to subclass the ListView and use the protected `getVirtualFlow()` method.

Reply:
We have many tests that access VirtualFlow, so that seems ok.

Reply:
would it be possible to update the JBS ticket with a valid code example?

Reply:
That would be very helpful.

Reply:
Oh, I didn't notice that I did upload the wrong file.
I've now uploaded the correct test application!